main.py

```diff
@@ -1,7 +1,45 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
 
 
 def main() -> QuerySet:
-    pass
+    genres_to_create = ["Western", "Action", "Dramma"]
+    for curr_genre in genres_to_create:
+        Genre.objects.create(
+            name=curr_genre
+        )
+
+    actors_to_create = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson"),
+    ]
+    for actor_data in actors_to_create:
+        Actor.objects.create(
+            first_name=actor_data[0],
+            last_name=actor_data[1]
+        )
```

Comment:
Please, use unpacking in the loop here to avoid indexing and make the code more descriptive